tests/headers/constify-module-enums-types.hpp

```diff
@@ -0,0 +1,38 @@
+// bindgen-flags: --constified-enum-module foo
+
+typedef enum foo {
+  THIS,
+  SHOULD_BE,
+  A_CONSTANT,
+  ALSO_THIS = 42,
+  AND_ALSO_THIS = 42,
+} foo;
+
+namespace ns1 {
+  typedef enum foo2 {
+    THIS,
+    SHOULD_BE,
+    A_CONSTANT,
+    ALSO_THIS = 42,
+  } foo2;
+}
+
+typedef foo foo_alias1;
+typedef foo_alias1 foo_alias2;
+
+typedef struct bar {
+  foo member1;
```

Comment:
Add the new option to `ns1::foo2`, and add an usage of it here, I'm pretty sure it'd be broken.